Utilities/DataFlow/test/test_SubframeUtils01.cxx

```diff
@@ -0,0 +1,56 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
The advantages of a framework are for instance that the program does not crash at the first failure and we get a precise list of passes/fails.